dbt_platform_helper/commands/environment.py

```diff
@@ -331,6 +251,21 @@ def get_cert_arn(session, env_name):
     raise click.Abort
 
 
+def get_env_ips(vpc: str, application_environment: Environment):
+    vpc_name = vpc if vpc else application_environment.account_id
```

Comment:
Under what circumstance would the VPC be named with an AWS account ID?

Reply:
I thought the var name was a misnomer here and referred to account name instead of ID, but had the wrong idea. Will amend